nsiqcppstyle_lexer.py

```diff
@@ -888,6 +888,10 @@ def validate_file(self, filename):
             f = open(filename)
             lines = f.readlines()
             f.close()
```

Comment:
Consider using a `with` outside the try

Reply:
This, and other file reads, now use "with".

Done.